pusher-api/user-guides/create-event-subscriptions.md

```diff
@@ -0,0 +1,175 @@
+
+1. Set up a test environment. For example, you can set up the environment using one of the following approaches:  
+    * You can use the destination URL that you have set up on your server.
+    * If you run a local server, you can make it publicly available using [ngrok](https://ngrok.com/).
+    * If you don't run a local server, you can use [Webhook.site](https://webhook.site) that provides an online view of all requests. <!-- how to treat 3rd party resources at Zettle? -->
+
+> **Disclaimer:** The Ngrok and Webhook.site are third-party resources that Zettle has no liability for the availability and intellectual property claims.
+
+2. Follow [Step 3: Create an event subscription](#step-3-create-an-event-subscription) to test the events and check the payloads.
+
+## Step 3: Create an event subscription
+You can subscribe to one or more events in one subscription request.
+
+1. Send a `POST` request to create event subscriptions. In the request body, `uuid` is the version 1 UUID that you generated in [Step 1: Generate a version 1 UUID](#step-1-generate-a-version-1-uuid).
+    
+    ```
+    POST /organizations/self/subscriptions
+   {
+     "uuid": "<version 1 UUID>",
+     "transportName": "WEBHOOK",
+     "eventNames": ["<event names>"],
+     "destination": "<URL to receive events>",
+     "contactEmail": "<email to receive notifications>"
+   }   
+    ```
+      
+    Example:
+    
+    The following example creates a subscription to event `ProductCreated` and `PurchaseCreated`.
+    ```
+        POST /organizations/self/subscriptions
+       {
+         "uuid": "ef64c5e2-4e16-11e8-9c2d-fa7ae01bbebc",
+         "transportName": "WEBHOOK",
+         "eventNames": [
+            "ProductCreated",
+            "PurchaseCreated"
+         ],
+         "destination": "https://yoururl.domain",
+         "contactEmail": "email_if_it_breaks@domain.com"
+       }   
+    ```
+    
+2. Check that the response returns with an HTTP status code `200 OK`.
+    * If yes, the subscription is created successfully.
+    * If no, update the `POST` request according to the error message. For more information on error messages, see [HTTP status code](../api-reference.md#createHttpStatusCode).
+    
+3. Save the value of `signingKey` from the response. This is the key used to sign all requests and should be stored so that you can validate the request. 
+
+    ```json
+    {
+        ...
+        "signingKey": "G6MWAEw1Fc6FWPkfiJiZ3j8Ya76I5ZbEDVPtzcPl6L6scsylmK5AEDyNyMe8N5cy"
+      }
+    ```
+4. If you use the destination URL to receive events for the first time, check that the server receives a test message.
+
+    Example:
+    
+    The example shows a test message.
+    ```json
+    {
+      "eventName" : "TestMessage",
+      "organizationUuid" : "ef64c5e2-4e16-11e8-9c2d-fa7ae01bbebc",
+      "messageId" : "0f674460-fab5-11e7-a310-0002ebd6a43c",
+      "payload" : { "data" : "payload" }
```

Comment:
As you noticed we always send payloads a Strings.
```suggestion
      "payload" : "{ \"data\" : \"payload\" }"
```

Reply:
Yes @Sarastro72, yes we were able to see only strings in the payloads when we tested. Thanks for confirming. We will fix these.